pallets/bridge-mapping/Cargo.toml

```diff
@@ -13,25 +13,14 @@ targets = ['x86_64-unknown-linux-gnu']
 [dependencies]
```

Comment:
I think we can get rid of this palle altogether, we currently do not use it and this was meant to be a generic way of supporting resources to actions for a different bridge setup.

Reply:
Ok, maybe it's better to do it in another PR, do you agree?

Reply:
Yep!